### DISCUSSION TITLE
saw something that reminded me of cosmos

### DISCUSSION THREAD

**Discussion by retrievaaaa** [CosmosOS/Cosmos#2813] [General]
https://hu.ma.ne/aipin/cosmos

now cosmos got a db AND an os conflicting with its name lmao

**Comment by glintpry** [comment]
you forgot the cosmos galaxy confliction tho

**Comment by glintpry** [comment]
and this
![image](https://github.com/CosmosOS/Cosmos/assets/83279568/4c9a80e9-8002-4392-903c-d2569fee8803)

**Comment by glintpry** [comment]
cosmos is also an astronomy survey btw, https://cosmos.astro.caltech.edu/
oh hey, theres an sdk too, https://github.com/cosmos/cosmos-sdk
and wait theres another, https://www.instagram.com/cosmos/?hl=en theres.... a wallpaper supplier cosmos
and i dont know what this is, but its cosmos too. https://thecosmosproject.org/
lastly, i think you can grow cosmos too, https://www.almanac.com/plant/cosmos

**Reply by zarlo** [reply]
there is Cosmos DevOps too [cosmos.dev](cosmos.dev)
